front/components/poke/workspace/table.tsx

```diff
@@ -7,6 +9,33 @@ import {
   PokeTableRow,
 } from "@app/components/poke/shadcn/ui/table";
 
+export function useCopyToClipboard(
```

Comment:
This guy already exist here:

https://github.com/dust-tt/dust/blob/c39e02f06cdc9dc99176c442e7d76b6d7452055a/front/components/misc/ContentBlockWrapper.tsx#L122

Reply:
Thanks, i overlooked it while searching (i thought it was in sparkle only).
Will move to a common place and use it.